postgres/osmap.yaml

```diff
@@ -29,9 +29,13 @@ RedHat:
   pkg_repo:
     name: pgdg{{ release }}
     humanname: PostgreSQL {{ repo.version }} $releasever - $basearch
-    baseurl: 'https://download.postgresql.org/pub/repos/yum/{{ repo.version }}/redhat/rhel-$releasever-$basearch'
     gpgcheck: 1
     gpgkey: 'https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-{{ release }}'
+   {%- if grains.os == 'Fedora' %}
```

Comment:
I think it's better to avoid whitespace suppression for Jinja here in YAML files. That could play tricks on you, and since such files are rendered internally, it is extremely hard to debug if something goes wrong.

Reply:
Okay, always open to suggestions - but what do you mean by whitespace suppression?  
To suppress whitespace, the statement could start at column 1, but I'm not suppressing whitespace since it starts at column 3.  Indentation is strong habit of mine since computer science 101, and I'm indenting the Jinja2 statements as per Jinja2 logical  flow, same way as any programming language. I'm not sure what to do here.

Reply:
I simply meant:
* indent with 4 (2+2) spaces at all changed lines (as YAML keys)
* replace `{%-` with `{%` cause it's not necessary

Reply:
I was unaware `{%-` mean whitespace suppression?  Well, I suspected but never bothered to check.  Sure no problem with these, thanks.